oppia/oppia-android
# Fix #1438 & #1761: Switch Profile Dialog lost in orientation change

## Explanation

Fix #1438 #1761

### How to test
1. open nav drawer
2. click on switch profile -> dialog appears
3. change orientation -> check if dialog visible or not

### How to review 
Check if the added test case works

## Checklist
